config/routes.rb

```diff
@@ -50,6 +50,10 @@
       end
     end
 
+    resources :targets, only: %i[update] do
```

Comment:
@bodhish: Shouldn't this just be `only: []` as we are not using `update` action for target anymore?